dev/tasks/crossbow.py

```diff
@@ -18,27 +18,26 @@
 # under the License.
 
 import re
-import sys
 import yaml
 import time
 import click
 import pygit2
+import github3
 import logging
 
 from enum import Enum
 from pathlib import Path
 from textwrap import dedent
 from jinja2 import Template
 from setuptools_scm import get_version
-from setuptools_scm.version import simplified_semver_version, meta
 
 
-logging.basicConfig(
-    level=logging.INFO,
-    format="[%(asctime)s] %(levelname)s Crossbow %(message)s",
-    datefmt="%H:%M:%S",
-    stream=click.get_text_stream('stdout')
-)
+# logging.basicConfig(
+#     level=logging.INFO,
+#     format="[%(asctime)s] %(levelname)s Crossbow %(message)s",
+#     datefmt="%H:%M:%S",
+#     stream=click.get_text_stream('stdout')
+# )
```

Comment:
Any reason not to just delete this code instead of leaving it here commented out?

Reply:
Well, the previous solution used logging instead of `click.echo`. It was Wes' request, but generated a lot of noise, so I've switched to directly printing to stdout. 
Honestly I'd remove it :)